src/Workspaces/Core/Portable/Workspace/IWorkspaceConfigurationService.cs

```diff
@@ -41,7 +41,8 @@ public DefaultWorkspaceConfigurationService()
         [property: DataMember(Order = 0)] StorageDatabase CacheStorage = StorageDatabase.SQLite,
         [property: DataMember(Order = 1)] bool EnableOpeningSourceGeneratedFiles = false,
         [property: DataMember(Order = 2)] bool DisableSharedSyntaxTrees = false,
-        [property: DataMember(Order = 3)] bool DeferCreatingRecoverableText = false)
+        [property: DataMember(Order = 3)] bool DeferCreatingRecoverableText = false,
+        [property: DataMember(Order = 4)] bool DisableRecoverableText = false)
```

Comment:
Do we still need DeferCreatingRecoverableText?

Reply:
yes.  two separate experiements.  one is if we have the concept at all.  the second is if we defer makign the recoverable text to a later point in time :)